test/specs/modules/Popup/Popup-test.js

```diff
@@ -192,6 +192,29 @@ describe('Popup', () => {
     })
   })
 
+  describe('keepInViewPort', () => {
+    it('will not alter the position and render outside the viewport if set to false', () => {
```

Comment:
Perfect!  We just need to add a test asserting that:
- `it('is enabled by default')`
- `it('alters the position when true')`